doc/api/buffer.markdown

```diff
@@ -468,7 +468,7 @@ additional performance that `Buffer.allocUnsafe(size)` provides.
 
 ### Class Method: Buffer.byteLength(string[, encoding])
 
-* `string` {String}
+* `string` {String|Buffer|DataView|ArrayBuffer}
```

Comment:
This doesn't explicitly mention the encompasing of all typed arrays. there a way that can be included?